### ISSUE TITLE
Question about metrics

### ISSUE THREAD

**Issue by TheLonelinessOfHS** [timeseriesAI/tsai#232]
Hi, sorry for this simple question but I wish someone could kindly tell me whether the metrics values shown during training are from the validation set or training set? I can see that the accuracy and precision of my model increased while the validation loss increased. If these values are from the training set, how can I get the validation scores?

**Comment by vrodriguezf** [comment]
By default, metrics in fastai are computed in the validation set. Are you
seeing that behaviour for many consecutive iterations or just for s few of
them?

Il ven 8 ott 2021, 02:06 TheLonelinessOfHS ***@***.***> ha
scritto:
